opencti-platform/opencti-front/src/private/components/nav/TopBar.tsx

```diff
@@ -194,6 +177,79 @@ const topBarQuery = graphql`
   }
 `;
 
+const routes = {
+  // ME
+  '/dashboard/profile/me': () => <TopMenuProfile />,
```

Comment:
We should think about path duplication : based direclty on route declaration.

And

We can have a TopBar service where we can push specific TopBar (like TopMenuProfile) when we are in a specific component to have dependency inversion.

Reply:
Let's think about that later 😆